testgres/node.py

```diff
@@ -430,6 +441,86 @@ def get_pid(self):
         # for clarity
         return 0
 
+    def get_child_processes(self):
+        ''' Returns child processes for this node '''
+
+        if psutil is None:
+            warnings.warn("psutil module is not installed")
```

Comment:
I think we should raise an exception here. The message may go unnoticed. Moreover, we could move this check to a small decorator.